shell/run_shell_cmd.go

```diff
@@ -14,29 +14,41 @@ import (
 
 	"github.com/gruntwork-io/terragrunt/errors"
 	"github.com/gruntwork-io/terragrunt/options"
+	"github.com/gruntwork-io/terragrunt/util"
 )
 
+var interactiveTerraformCommands = []string{
+	"console",
```

Comment:
Is interactive the right term here? E.g., if you run `terraform apply`, it interactively prompts you for a "yes" to confirm deployment. That part works just fine with Terragrunt as-is. So is t more that we need a REPL? Or even more explicitly, `terraformCommandsThatNeedPty`?

Reply:
Good point. Renamed! 9df6643